src/pages/Schedule.js

```diff
@@ -0,0 +1,314 @@
+import React from "react"
+import Helmet from "react-helmet"
+import {
+    Navbar, 
+    Footer
+} from "../components"
+import s from "styled-components"
+import { Title, Text, Subtitle, Link } from "../components/shared/Typography"
+import Button from '../components/shared/Button'
+import {GREEN} from "../utils/constants.js"
+import Dennis from '../images/dennis woodside.jpg'
+import Lidiya from '../images/lidiya-dervisheva.jpeg'
+import Seth from '../images/seth.jpg'
+import Marissa from '../images/marisa sweeney.jpg'
+import Sam from '../images/sam bursten.jpeg'
+
+const Headshot = s.img`
+    width: 13vw;
+    height: 13vw;
+    object-fit: cover;
+    border-radius: 50%;
+    margin: 1vw;
+`
+
+const Columns = s.div`
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    margin-top: 2vw;
+`
+const Caption = s.figcaption`
+    font-family: roboto;
+    font-weight: bold;
+    font-size: 20px;
+`
+
+const EventGroup = s.div`
+    margin: 2vw 0;
+`
+
+const EventNoDate = s.div`
+    border-left: 0.3rem solid ${GREEN};
+    padding: 0vw 1vw 0vw;
+    margin: 1vw 2vw;
+`
+
+const EventWithDate = s.div`
+    margin: 1vw 0;
+`
+
+const EventWithDateInfo = ({date, description, btext, blink}) => {
+    // var day = /^\d{1,2}\/\d{1,2}/;
+    var day = /\s\d{1,2}\s/;
+    var month = /^\w{3}/;
+    var time = /(\d{1,2}:\d{1,2}|\d{1,2})\sPM/;
+
+    return (
+        <div>
+            <table css={`margin: 0.5vw 2vw;`}>
+                <tbody>
+                <tr css={`border-left: 0.3rem solid ${GREEN}`}>
+                    <td css={`width: 7vw; padding-left: 1vw`}>
+                        <div css={`text-align: center`}>
+                            <Text fontSize="0.3rem" roboto bold>{date.match(month)}</Text>
+                            <Text fontSize="1.5rem" roboto bold css={`line-height: 1`}>{date.match(day)}</Text>
+                            <Text fontSize="0.3rem" roboto bold >{date.match(time)[0]}</Text>
+                            <Text roboto bold css={`line-height: 0.8`}>EST</Text>
+                        </div>
+                    </td>
+                    <td css={`padding-left: 1vw`}>
+                        <EventDescription description={description}/>
+                        {btext && <div><br/><Button href={blink}>{btext}</Button></div>}
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+        </div>
+    )
+}
+
+const EventDescription = (props) => {
+    var linkRegex = /https?:\/\/.*$/;
+
+    return(
+        <Text css={`white-space:pre-line`}>
+            {(props.description).map((str, i) => {
```

Comment:
you can pass JSX as props, so might be better to write something like

`description={<><elem>button</elem><title>title</title></>}`

etc. By computing on a list of strings, you are essentially building a templating language when you are already working with React, which is a templating language by virtue of integrating with JSX...

Reply:
Ignore how the description is whitespaced - this was the only way I could get MD to not interpret the elements as actual honest to god html to format :/